src/genie/libs/parser/iosxe/tests/ShowVrrp/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = { 
+     {
+    "interface": {
+        "GigabitEthernet1": {
+            "group_number": {
+                "Group 100": {
+                    "auth_text": '"hash"',
+                    "device_advertise_interval": "1.000",
+                    "master_adv_int": "1.000",
+                    "master_down_int": "3.531",
+                    "master_ip_address": "192.168.1.233",
+                    "master_router": "local",
+                    "master_router_priority": "120",
+                    "preemption_state": "enabled",
+                    "priority": "120",
+                    "track_object_decrement_value": 30,
+                    "track_object_number": "1",
+                    "track_object_state": "Up",
+                    "virtual_ip_address": "192.168.10.1",
+                    "virtual_mac_address": "0000.5e00.0164",
+                    "vrrp_group_name": "DC-LAN " "Subnet",
+                    "vrrp_state": "Master",
```

Comment:
we already knew this output from vrrp command, so simply have `state`.
```suggestion
                    "state": "Master",
```

Reply:
confirmed the change